GitCommands/Git/GitModule.cs

```diff
@@ -1354,6 +1354,30 @@ public string ResetFile(string file)
             });
         }
 
+        public string ResetFiles(IReadOnlyList<string> files)
+        {
+            if (files.Count == 0)
+            {
+                return string.Empty;
+            }
+
+            var sb = new StringBuilder();
+            var arguments = new GitArgumentBuilder("checkout-index")
+                {
+                    "--index",
+                    "--force",
+                    "--"
+                }
+                .BuildBatchArguments(files.Select(item => item.ToPosixPath().Quote()));
+
+            foreach (var arg in arguments)
+            {
+                sb.Append(_gitExecutable.GetOutput(arg.Argument));
+            }
+
+            return sb.ToString();
```

Comment:
Could the above loop be made an extension method on `_gitExecutable` similar to `RunBatchCommand`, something like:

```
            return _gitExecutable.GetBatchedOutput(batches);

Reply:
Added `GetBatchOutput` extension method. Is `ed` in `Batch` required? I'm not very good with grammar (Should have Grammarly Extension for Visual Studio).